src/scss/login.module.scss

```diff
@@ -279,11 +284,379 @@ $white: #fff;
 }
 
 .button--github {
-  color: #333;
+  color: #3c4b64;
   line-height: 46px;
   margin-bottom: 3rem;
 
   &:hover {
     color: $tiber;
   }
 }
+.btnSlideEffect:hover,
+.btnSlideEffect:focus {  
+  animation: pulse 1s;
+  background-color: #3c4b64;
+  color:#ebedef;
+  box-shadow: 0 0 0 2em rgba(#3c4b64,0);
+}
+
+// /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Landing Page Sections CSS
+// Section Container
+.slide{
+  height:100%;
+  position: relative;
+  padding:1em;
+  font-size:14px;
+  margin-top:8em;
+  }
+
+ 
+  // Feature Section
+  .Features
```

Comment:
Let's ensure all classes use small case and hyphens for naming. Change this to `features` and same for other affected classes.